### DISCUSSION TITLE
Reopened and answered

### DISCUSSION THREAD

**Discussion by setchy-test** [setchy/gitify-notifications-sandbox#26] [Q&A] [answered]
Reopened and answered

**Comment by setchy-test** [comment, marked as answer]
Answer

**Comment by setchy-test** [comment]
Reopen

**Comment by setchy** [comment]
replying to see if this changes participation attributes

**Reply by setchy** [reply]
Tagging to see if this changes participation attributes @setchy

**Reply by setchy-test** [reply]
Tagging to see if this changes participation attributes @setchy
